--- a/data/org.gnome.Documents.appdata.xml.in	2019-03-11 21:52:06.000000000 +0100
+++ b/data/org.gnome.Documents.appdata.xml.in	2019-09-13 12:38:18.532651713 +0200
@@ -47,7 +47,10 @@
     <screenshot>
       <image type="source">https://gitlab.gnome.org/GNOME/gnome-documents/raw/master/data/screenshots/gnome-documents-3.png</image>
     </screenshot>
-  </screenshots>
+  </screenshots>  
+  <releases>
+    <release version="3.32.0" date="2019-03-11" />
+  </releases>
   <project_group>GNOME</project_group>
   <update_contact>debarshir_at_gnome.org</update_contact>
   <translation type="gettext">gnome-documents</translation>
